octavia-cli/octavia_cli/apply/resources.py

```diff
@@ -357,6 +369,15 @@ def update_payload(self):
             name=self.resource_name,
         )
 
+    def _get_comparable_configuration(self):
+        """Get the object to which local configuration will be compared to.
+
+        Returns:
+            dict: Remote source configuration.
+        """
+        comparable_configuration = super()._get_comparable_configuration()
+        return comparable_configuration.connection_configuration
```

Comment:
This can't be done in BaseResource and only be overwritten by Connection object?

Reply:
I duplicated this on purpose as this is really specific to source and destination and not something common to the other potential resources. I was thinking of adding a parent class for `Source` and `Destination` (`SourceOrDestination` 🤔 ) but thought it was too much for this PR.